interfaces/python_sdist/pyproject.toml.in

```diff
@@ -22,19 +22,18 @@ classifiers = [
     "Programming Language :: Cython",
     "Programming Language :: Fortran",
     "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
     "Programming Language :: Python :: Implementation :: CPython",
     "Topic :: Scientific/Engineering :: Chemistry",
     "Topic :: Scientific/Engineering :: Physics",
 ]
 requires-python = "@py_requires_ver_str@"
 dependencies = [
-    "numpy >= 1.12.0,<3",
-    "ruamel.yaml >= 0.15.34",
+    "numpy >= 1.21.0,<3",
+    "ruamel.yaml >= 0.17.16",
```

Comment:
These could probably be templated from SConstruct. I'm not sure why I wouldn't have done that in the first place 🤔

Reply:
These versions are set directly on the SCons environment, which isn't even created in the case of the `sdist` target. I'm sure it could be fixed, but I think I'll just leave it as is for now, with a comment that the values in `pyproject.toml.in` also need to be updated alongside the ones int `SConstruct`.